src/pages/landing/ContentSection/QuotesCarousel.tsx

```diff
@@ -0,0 +1,256 @@
+import { useState } from 'react';
+import { Carousel } from 'react-responsive-carousel';
+import { styled } from 'styled-components';
+import { FONT_MEDIUM, MOBILE_MAX_WIDTH } from '~/components';
+import QUOTES_DATA from '~/data/quotes.json';
+
+// simulate circular carousel
+const QUOTES = [...QUOTES_DATA, ...QUOTES_DATA, ...QUOTES_DATA];
```

Comment:
have you tried using the `infiniteLoop` property?

I think it might be better than they repeated themselves and came to an end

Reply:
The thing is, the carousel does not start from the first quote by design.
With just the infiniteLoop, the user has to go left to read the first quote and then right, or go to the end by clicking right and then the carousel jumps to the first. It's weird.

That's why I simulated a circular a carousel. But if we can live with that, it's okay haha

Reply:
[b683dea](https://github.com/defi-wonderland/web/pull/113/commits/b683deae12b7327c1a61a0bf2b5227c7c2c066f7)